content/blockchain-contracts/ink-workshop/slides.md

```diff
@@ -8,7 +8,9 @@ duration: 20 min
 
 Notes:
 
-This workshop was an idea that came from first wave of PBA in Cambridge. The first time it was ever played was back in Buenos Aires. It's very much a PBA right of passage, so it's exciting to be able to play today.
+This workshop was an idea that came from first wave of PBA in Cambridge.
+The first time it was ever played was back in Buenos Aires.
+It's very much a PBA right of passage, so it's exciting to be able to play today.
```

Comment:
1SPL rule applied in many places here, seems to not break intended rendering :crossed_fingers: